src/rascal/math/utils.hh

```diff
@@ -189,6 +189,39 @@ namespace rascal {
 
       return rel_diff;
     }
+
+    /**
+     * Routine to compute the relative difference between 2 double.
+     * The return value depend on the closeness to zero:
+     * + if ref and test are not zero, then return relative error
```

Comment:
I am not sure to follow you here. Are you referring to the third point in this comment ?

Reply:
No, I'm referring to the fourth case that's not mentioned (ref is nonzero, but test is zero). I'm assuming that's meant to be covered under the first case, so I would reword it to just "if ref is nonzero, then return relative error" (because that's what the function does).